packages/node/src/lib/object-to-array.ts

```diff
@@ -18,11 +23,11 @@ export function objectToArray(
     }
 
     return prev;
-  }, []);
+  }, [] as { name: string; value: unknown }[]);
```

Comment:
Might make sense to type `{name: string; value: unknown}` as an interface? It's used a bunch of times here